regression/tests/studio/test_course_update.py

```diff
@@ -29,47 +29,102 @@ def setUp(self):
             self.course_info['run']
         )
         self.course_update_page.visit()
+        self.course_update_content_selector = '#course-update-list li' \
+                                              ' .post-preview .update-contents'
+        self.course_update_text = 'New update:{}'.format(uuid4().hex)
 
-    def test_course_update(self):
+    def tearDown(self):
         """
-        Verifies creation, editing and deletion of a course update
+        Delete the courses created.
+
+        It is called after every test run irrespective of
```

Comment:
This particular docstring is not necessary. It's also not entirely accurate; if there is an error, I believe `tearDown` gets skipped; that's one of the reasons I like `addCleanup`.